lib/checkPullRequestTemplate.js

```diff
@@ -203,34 +252,40 @@ const checkTemplate = async (context) => {
 
   const explanationMessage = checkPRExplanation(pullRequest);
   const checklistMessage = checkEssentialChecklist(pullRequest);
+  const proofOfChangesMessage = checkProofThatChangesAreCorrect(pullRequest);
   let commentBody = '';
-  if (explanationMessage && checklistMessage) {
-    commentBody =
-      'Hi @' +
-      pullRequest.user.login +
-      ', ' +
-      explanationMessage +
-      '<br>Also, ' +
-      checklistMessage.trim() +
-      ' Thanks!';
-  } else if (explanationMessage) {
-    commentBody =
-      'Hi @' +
-      pullRequest.user.login +
-      ', ' +
-      explanationMessage +
-      ' Thanks!';
-  } else if (checklistMessage) {
-    commentBody =
+
+  count = 1;
+
+  commentBodybasic = commentBody =
       'Hi @' +
       pullRequest.user.login +
-      ', ' +
-      checklistMessage.trim() +
-      ' Thanks!';
-  } else {
+      ', can you complete the following:\n';
+
+  if (explanationMessage) {
+    commentBody += count.toString() +
+    '. ' + explanationMessage + '\n';
+    count += 1;
+  }
+
+  if (checklistMessage) {
+    commentBody += count.toString() +
+    '. ' + checklistMessage + '\n';
+    count += 1;
+  }
+
+  if (proofOfChangesMessage) {
+    commentBody += count.toString() +
+    '. ' + proofOfChangesMessage + '\n';
+  }
+
+  if (commentBody === commentBodybasic){
     return;
   }
 
+  commentBody += 'Thanks!';
+
+  commentBody = commentBody.trim();
```

Comment:
Can we extract this comment generation into a different function so we can have separate unit tests for it?

Reply:
DONE. After confirmation, I'll add unit tests